src/components/Profile/ProfileStyled.js

```diff
@@ -1,347 +1,65 @@
-  padding: 0;
-  margin: 0;
-`;
-
-export const ImgBtn = styled.img`
-  width: 14px;
-  height: 14px;
-
-  cursor: pointer;
-
-  @media screen and (min-width: 768px) {
-    width: 24px;
-    height: 24px;
-  }
-`;
-
-export const LabelImg = styled.label`
-  position: absolute;
-  left: 43px;
-  top: 62px;
-  @media screen and (min-width: 768px) {
-    left: 76px;
-    top: 106px;
-  }
-`;
-
-export const ContainerImg = styled.div`
-  position: absolute;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-
-  left: 115px;
-  top: -30px;
-
-  width: 72px;
-  height: 72px;
-
-  border: 2px solid var(--accent-background-color);
-  border-radius: 50%;
-  background: var(--main-background-color);
-
-  @media screen and (min-width: 768px) {
-    position: relative;
-    margin-top: 40px;
-    margin-bottom: 14px;
-
-    left: 0px;
-    top: 0px;
-
-    width: 124px;
-    height: 124px;
-  }
-  @media screen and (min-width: 1440px) {
-    margin-top: 60px;
-    margin-bottom: 20px;
-  }
-`;
-
-export const ImgAvatar = styled.img`
-  width: 100%;
-  height: 100%;
-
-  border-radius: 50%;
-`;
-
-export const SvgAvatar = styled.svg`
-  width: 48px;
-  height: 48px;
-
-  border-radius: 50%;
-`;
-
-export const VectorPng = styled.svg`
-  position: absolute;
-
-  transform: rotate(270deg);
-
-  stroke: var(--title-text-main-color);
-  fill: none;
-
-  cursor: pointer;
-
-  left: 260px;
-  bottom: 175px;
-
-  width: 18px;
-  height: 18px;
-  @media screen and (min-width: 768px) {
-    left: 310px;
-    bottom: 219px;
-  }
-  @media screen and (min-width: 1440px) {
-    left: 314px;
-    bottom: 118px;
-  }
-`;
-
-export const LabelBtn = styled.label`
-  display: flex;
-  flex-direction: column;
-  font-family: 'Inter', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 14px;
-  line-height: calc(18 / 14);
-
-  color: var(--label-text-color);
-
-  @media screen and (min-width: 768px) {
-  }
-  @media screen and (min-width: 1440px) {
-  }
-`;
-
-export const Btn = styled.button`
-  margin-left: auto;
-  margin-right: auto;
-  margin-top: 40px;
-  margin-bottom: 40px;
-
-  padding: 0;
-
-  width: 195px;
-  height: 46px;
-
-  border-radius: 16px;
-  border: none;
-
-  cursor: pointer;
-
-  transition: 250ms cubic-bezier(0.4, 0, 0.2, 1);
-
-  font-family: 'Inter', sans-serif;
-  font-style: normal;
-  font-weight: 600;
-  font-size: 14px;
-  line-height: calc(18 / 14);
-
-  background: var(--accent-background-color);
-  color: var(--btn-text-color);
-
-  @media screen and (min-width: 768px) {
-    width: 262px;
-    height: 48px;
-    margin-top: 22px;
-    margin-bottom: 40px;
-  }
-
-  @media screen and (min-width: 1440px) {
-  }
-
-  &:hover,
-  :focus {
-    cursor: pointer;
-    background-color: #2b78ef;
-  }
-  &:active {
-    filter: blur(0.1rem);
-  }
+export const EditButton = styled.button`
+width: 18px;
+height: 18px;
+background-image: url(${props => props.$icon === 'edit' ? PencilIcon : CloseIcon});
```

Comment:
Lets try to do is this way `background-img: url(${({ $icon }) => ($icon ? PencilIcon : CloseIcon)}); `

Reply:
I am providing conditionally variable $icon into component:
<EditButton
          onClick={editMode ? cancelEdit : toggleEditMode}
          $icon={editMode ? 'close' : 'edit'}
        />
        
and compare check if it is 'edit' or not to set appropriate background-image image.